backend/build.gradle

```diff
@@ -108,7 +126,6 @@ dependencies {
     runtimeOnly 'com.azure:azure-core-http-netty:1.11.9'
     runtimeOnly 'com.azure:azure-core:1.27.0'
     runtimeOnly 'com.azure:azure-storage-common:12.15.1'
-    runtimeOnly "com.azure:azure-storage-queue:12.12.1"
```

Comment:
why is this dependency being removed? I know we still use the azure storage queue